ssl/statem/extensions_clnt.c

```diff
@@ -952,8 +952,11 @@ EXT_RETURN tls_construct_ctos_padding(SSL_CONNECTION *s, WPACKET *pkt,
              * Add the fixed PSK overhead, the identity length and the binder
              * length.
              */
+            int tmpsize = EVP_MD_get_size(md);
+            if (tmpsize < 0)
```

Comment:
`<= 0` as 0 length is also not possible. And please add an empty line above.

Reply:
Thanks. I have force-pushed a new commit to fix the problem.

Reply:
@JiangJias Could you please change similarly all the places where you check the md size?

Reply:
Fine